setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
I renamed these fixtures to `db_connection` and `db_session` to make them less generic.